packages/server/src/stacks-controller.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
```

Comment:
@zone117x is this the functionality you were talking about? We were looking to put stuff like this in sidecar, too!